docs/content/documentation/content/linking.md

```diff
@@ -22,23 +22,26 @@ You can also manually specify an id with a `{#…}` suffix on the heading line:
 # Something manual! {#manual}
 ```
 
-This is useful for making deep links robust, either proactively (so that you can later change the text of a heading without breaking links to it) or retroactively (keeping the slug of the old header text, when changing the text). It can also be useful for migration of existing sites with different header id schemes, so that you can keep deep links working.
+This is useful for making deep links robust, either proactively (so that you can later change the text of a heading
+without breaking links to it) or retroactively (keeping the slug of the old header text when changing the text). It
+can also be useful for migration of existing sites with different header id schemes, so that you can keep deep
+links working.
 
 ## Anchor insertion
-It is possible to have Zola automatically insert anchor links next to the heading, as you can see on the site you are currently
-reading if you hover a title.
+It is possible to have Zola automatically insert anchor links next to the heading, as you can see on this site
```

Comment:
site -> documentation?

Reply:
I see. "site your are currently reading" = "this documentation". Please make this change.